core/iwasm/interpreter/wasm_interp_fast.c

```diff
@@ -3536,6 +3541,24 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
                 HANDLE_OP_END();
             }
 
+#if WASM_ENABLE_SIMD != 0
```

Comment:
I know it's repetitive but let's do it in a separate PR (same for all the places where v128 repeats i64 a lot